src/main/java/com/sparta/gamjaquick/menu/entity/Menu.java

```diff
@@ -41,25 +51,24 @@ public class Menu extends AuditingFields {
 
 
     public Menu(UUID storeId, MenuRequestDto menuRequestDto) {
-        this.storeId = storeId;
+        this.store = new Store(storeId);
```

Comment:
매개변수로 storeId를 받아와서 new Store(storeId)를 할 때 storeId가 null 인 경우 예외가 발생하기 때문에 Menu를 생성할 때 Store 객체를 매개변수로 받아서 그 객체를 넣는 것도 방법이 될 것 같습니다!
```suggestion
public Menu(Store store, MenuRequestDto menuRequestDto) {
        this.store = store;
```